Lib/glyphsets/test_strings.json

```diff
@@ -1,35 +1,35 @@
 {
-    "GF_Latin_Kernel": {
+    "GF_Latin_Core": {
         "Letter Groups": [
-            "EFHIJLTU", 
-            "AVWXZ",
-            "KMNY",
-            "COQS",
-            "BPRDG",
-            "fijltr",
-            "kvwxyz",
-            "hmnu",
-            "ceogs",
-            "abdpq"
+            "EFHIJLTU AVWXZ KMNY",
+            "COQS BPRDGÞẞ",
+            "hmnu ceogðsß abdpqþ",
+            "kvwxyz fijltr",
+            "0123456789",
+            ".,:;/|?!-–—&@%([{}])°",
+            "+−×÷=≠><≥≤±≈~¬∞"
         ],
         "Spacing": [
             "HHOHHOOHOO HIHOIO",
             "uuonouonoo ninoioolonlniuo",
-            "010 080 030 020 050 060 070"
+            "010 080 030 020 050 060 070",
+            "3+4=7 5>2 2x²−(8×7)÷9≈1",
+            "ĿL·L ŀl·l"
```

Comment:
Similar to minute and second, we don't have Ldot 013F or ldot 0140 included in the Latin Core.
So we should add them.

Reply:
yeah I don't know how to do with this one. Monospace font would do with Ldot, while the other should do with localised middle dot

Reply:
Are these above
Well, in that case, we should probably 
- Add the localized middle dot to this sample (I saw your attempt to do that in a previous commit, did you undo that because of the monospace fonts?)
    - Alternatively, make the `diffbrowsers_proofer.html` to use the localised glyph if present in the font so we don't receive tofus in QA results. Similar to what it seems `diffbrowsers_text.html` does. We have a `rebel·lió ` word in the latter, and it works to shape the correct glyph. 
- Add a FB check to ensure monospace letters include Ldot ones.

Reply:
Yep! my attempt failed cause the json didn't seem to like `<span lang=cat>` `</span>`. @m4rc1e how can we localise this string to avoid tofu in the proofer when the the Ldot codepoint is missing?

Reply:
Unfortunately the current implementation only works as strings. Perhaps I can allow us to add language tags as well?

Reply:
may be useful of we extend the proofer to more languages

Reply:
<img width="1552" alt="Screenshot 2023-11-06 at 11 26 50" src="https://github.com/googlefonts/glyphsets/assets/7525512/af394a2b-cb27-4c8c-b603-57f1e6ffab55">

I'm updating the test_strings.json file so you have to provide a lang as well for each string.